<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResetPWWindow</class>
 <widget class="QMainWindow" name="ResetPWWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>610</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>610</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>610</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ResetPassword</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.ico</normaloff>logo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line_separator">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_resetpw">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>重設密碼</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_code">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>驗證碼:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_username">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>使用者名稱:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_resetinstru">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>721</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;請輸入您的使用者名稱，以再次確認您的身分&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pw1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>新密碼:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pw2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>430</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>再次輸入新密碼:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pwinstru">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>731</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#616161;&quot;&gt;密碼要求: 僅接受英數，長度至少8字元，包含ˊ一個大寫、一個小寫、一個特殊字元 (#$%!^&amp;amp;*~`&amp;quot;/,.) &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_submit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>500</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MV Boli&quot;;
background-color: rgb(226, 226, 226);</string>
    </property>
    <property name="text">
     <string>送出</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_username">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>250</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pw1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>370</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pw2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>430</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_code">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>310</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_resendcode">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>500</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MV Boli&quot;;
background-color: rgb(226, 226, 226);</string>
    </property>
    <property name="text">
     <string>重寄驗證碼</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_lang">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
